Add a description about RLS to IVM documentation.

yugo-n opened the pull request:
Github issue #71

yugo-n committed:
Add a description about RLS to IVM documentation.

Github issue #71

yugo-n committed:
Fix some white spaces

tatsuo-ishii commented on doc/src/sgml/rules.sgml:
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
+<sect2>
+<title>Row Level Security</title>
+<para>
+    If some base table have row level security policy, rows that are not invisible
```
"base table" should be "base tables".
"invisible for" should be "invisible to".

tatsuo-ishii commented on doc/src/sgml/rules.sgml:
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
+<para>
+    If some base table have row level security policy, rows that are not invisible
+    for the materialized view are excluded from the result. In addition, such rows
+    are excluded also when views are incrementally maintained. However, if a new
```
"also" should "as well" or "too".

tatsuo-ishii commented on doc/src/sgml/rules.sgml:
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
+    If some base table have row level security policy, rows that are not invisible
+    for the materialized view are excluded from the result. In addition, such rows
+    are excluded also when views are incrementally maintained. However, if a new
+    policy is defined  or policies are changed after materialized view definition,
```
There's extra space after "defined".
"after materialized view definition" should be "after the materialized view definition was created"

tatsuo-ishii commented on doc/src/sgml/rules.sgml:
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
+    for the materialized view are excluded from the result. In addition, such rows
+    are excluded also when views are incrementally maintained. However, if a new
+    policy is defined  or policies are changed after materialized view definition,
+    the new policy will not applied to the view contents. To apply the new policy,
```
"will not applied" should be "will not be applied".

yugo-n commented:
@tatsuo-ishii Thank you so much for your many pointing out my mistake!  I'll fix them.

yugo-n committed:
Fix typos and add a missing word

yugo-n committed:
Add a word forgotten in the pervious commit

yugo-n commented:
@tatsuo-ishii 

I fixed the typos. Also, I added a missing word as following:

"not invisible to the materialized view" 
 -> "not invisible to the materialized view's owner"

yugo-n commented:
Thanks for your review.  I'll merge it.
